examples/uid_higher_or_equal_than_zero.yaml

```
@@ -0,0 +1,8 @@
+name: uid_higher_or_equal_than_zero
+description: This policy traces read and write syscall for uids higher or equal than 0
```

```suggestion
name: uid_equal_to_or_higher_than_zero
description: This policy traces read and write syscall for uids equal to or higher than 0
```
rename the file name as well.

---

why? if the signal is `>=` higher or equal